### PR TITLE
Draft system distributor doc

### PR DESCRIPTION
**Pull request by bradleycamacho**
For NR-32275

### CODE REVIEW

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+[Default polling information](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-polling-intervals-infrastructure-integrations) for the AWS integrations:
+- New Relic polling interval: 5 minutes
+- Amazon CloudWatch data interval: 1 minute or 5 minutes
+
+
+## Find and use data
+To find your integration data, go to your account on [one.newrelic.com ](one.newrelic.com).
+
+For metrics:
+On the left column select **Data explorer > Metrics** and filter by `aws.???????`
```
Comment (bradleycamacho) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx:50]: @4demos are there metrics we can filter for this integration?

Comment (4demos) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx:50]: I'd refer here to our Infrastructure Agent page. SMD is just a way to install our Infrastructure Agent. After it's installed the rest is the same regardless the way it has been installed

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+You can change the polling frequency and filter data using [configuration options](/docs/integrations/new-relic-integrations/getting-started/configure-polling-frequency-data-collection-cloud-integrations).
+
+[Default polling information](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-polling-intervals-infrastructure-integrations) for the AWS integrations:
+- New Relic polling interval: 5 minutes
+- Amazon CloudWatch data interval: 1 minute or 5 minutes
+
+
+## Find and use data
+To read more about how to find your data, see our [Introduction to AWS integrations](/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations/#insights)
```
Comment (bradleycamacho) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx:47]: How's this in that case @4demos

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+title: AWS System Manager Distributor
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS System Manager Distributor monitoring integration: what data it reports, and how to enable it."
```
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx:7]: actually AWS System Manager Distributor is a way to provide our infrastructure agent to new and old customers. It's a way of easier installation in a diverse environment. It's very different from AppRunner integration. We do not make any monitoring of SMD, no any extra data, no way to enable it. The point of sale here is to install our infrastructure agent on a fleet of different in OS and architecture instances by a single click. Monitoring goes only for the instance, the agent is installed on and by the way the infra agent does so, noth
ing added, nothing removed

Comment (bradleycamacho) [reply src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx:3]: Ah interesting! I just took another pass to clarify this

**Comment by bradleycamacho** [comment]
@4demos how does this look with the new edits? Feel free to make direct suggestions on wording.

**Comment by bradleycamacho** [comment]
@4demos also, can you clarify if we even call this an integration?

File: src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx [hunk 4]
```diff
@@ -1,20 +1,16 @@
...
 title: AWS System Manager Distributor
-tags:
-  - Integrations
-  - Amazon integrations
-  - AWS integrations list
 metaDescription: "New Relic's AWS System Manager Distributor monitoring integration: what data it reports, and how to enable it."
 ---
 
-[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for reporting your application and AWS System Manager Distributor data to New Relic observability platform. This document explains the integration's features, how to activate it, and what data can be reported.
+Quickly integrate your AWS instances with New Relic's infrastructure agent through our integration for AWS System Manager Distributor. This document explains the integrations features, how to activate it, and what data can be reported.
```
Comment (4demos) [inline src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx:6]: I'd say:
"Quickly install  New Relic's infrastructure agent on your AWS instances through our integration for AWS System Manager Distributor."

**Pull request merged**